tests/op/test_run_nvnmd_train.py

```
@@ -0,0 +1,796 @@
+        config = self.config.copy()
+        config["init_model_policy"] = "no"
+
+        task_path = self.task_path
+        Path(task_path).mkdir(exist_ok=True)
+        with open(Path(task_path) / train_script_name, "w") as fp:
+            json.dump(self.idict_v2, fp, indent=4)
+        task_name = self.task_name
+        work_dir = Path(task_name)
+
+        ptrain = RunNvNMDTrain()
+        with self.assertRaises(FatalError) as ee:
```

Ruff (0.11.9)

Local variable `ee` is assigned to but never used

Remove assignment to unused variable `ee`

(F841)